src/server/createProvider.tsx

```diff
@@ -22,6 +22,7 @@ const createProvider = async (sdkKey: string, user: LDUser) => {
         value={{
           flags,
           ldClient,
+          key: user.key,
```

Comment:
Can we please pass the LDUser object here rather than just the user key? That will be useful in the future for other use cases.

Reply:
Sure, I can modify for that.